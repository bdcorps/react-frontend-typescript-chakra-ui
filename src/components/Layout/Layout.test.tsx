import { shallow } from 'enzyme';
import { Layout } from '.';

describe('Hello, Enzyme!', () => {
    it('renders', () => {
        const wrapper = shallow(
            <Layout>
                <h1>Hello</h1>
            </Layout>
        );

        expect(wrapper.find('h1').html()).toMatch(/Hello/);
    });

    it('renders snapshots, too', () => {
        const wrapper = shallow(
            <Layout>
                <h1>Hello</h1>
            </Layout>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
