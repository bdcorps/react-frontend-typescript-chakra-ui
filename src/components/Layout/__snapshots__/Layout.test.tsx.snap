// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello, Enzyme! renders snapshots, too 1`] = `
<Box>
  <Flex
    direction="column"
    m="2rem auto"
    maxW={
      Object {
        "xl": "1200px",
      }
    }
  >
    <Container
      maxW="container.xl"
    >
      <h1>
        Hello
      </h1>
    </Container>
  </Flex>
</Box>
`;
